_posts/2022-12-26-c++-sanitizers.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+title: C++ Sanitizers
+date: 2022-12-26
+category: cpp
+tags:
+- cpp
+redirect_from:
+- /cpp/2022/12/26/cpp-sanitizers/
+- /cpp-sanitizers.html
+---
+
+Hi, there! I recently learned about C++ Sanitizers. That's an amazing concept.
+C++ Sanitizers are recently added in Clang and GCC compilers for runtime code analysis.
+Sanitizers detect errors that are usually not detected during compile time.
```

Comment:
See https://github.com/khushi-411/khushi-411.github.io/commit/c48ef84e284ce7d52de6434139bc3a0317916222#r94216338

Reply:
Updated this: https://github.com/khushi-411/khushi-411.github.io/commit/c48ef84e284ce7d52de6434139bc3a0317916222#r94231030.